### ISSUE TITLE
laminas-feed still relevant for this component?

### ISSUE THREAD

**Issue by froschdesign** [laminas/laminas-cache#282]
### Bug Report

|    Q        |   A
|------------ | ------
| Version(s)  | 3 & 4

#### Summary

laminas-feed is listed as required package for developing but is this really need?

Branch 4.0.x:

https://github.com/laminas/laminas-cache/blob/2db8ef0c511b9cd358a92f8ea6626d47cd4b16df/composer.json#L52

Branch 3.13.x:

https://github.com/laminas/laminas-cache/blob/66cf9a209163cadc65d93483667b995c5ead1563/composer.json#L56

**Comment by boesing** [comment]
I'd say no. I guess that was the case due to autocompletion in documentation. if laminas-feed is not mentioned in the docs anymore, we can remove that dependency.
if it is, we might want to think about moving the documentation to the feed component instead.

**Comment by Xerkus** [comment]
There is no mention of feed in the package whatsoever:
```
[laminas-cache (3.11.x)]$ ag feed
composer.json
56:        "laminas/laminas-feed": "^2.20",

composer.lock
<snip>
```

Adapters also require laminas-feed but they have no occurrences of "feed" in either source or tests.

**Comment by froschdesign** [comment]
> if laminas-feed is not mentioned in the docs anymore, we can remove that dependency.

Correct, there was an example in the documentation of version 2:

https://github.com/laminas/laminas-cache/blob/2.13.x/docs/book/pattern/class-cache.md#examples

> if it is, we might want to think about moving the documentation to the feed component instead.

The example is for the removed class cache feature, so it is no longer needed.
(But the description is missed in version 2 of the laminas-cache documentation. 🤔)

@Xerkus @boesing 
Thanks for your answers!

**Comment by froschdesign** [comment]
Fixed with #291
